global.json

```diff
@@ -3,13 +3,18 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "6.0.100-rc.1.21458.32",
+    "dotnet": "6.0.100",
+    "runtimes": {
+      "dotnet/x64": [
+        "2.1.7"
+      ]
```

Comment:
No wait I deleted this, why is it back?

Reply:
I re-deleted it, but I think it was there when this PR was made. I'm surprised it didn't disappear with the "clean" merge.